modules/opentelemetry-otlp-http4s-grpc-exporter/src/main/scala/trace4cats/opentelemetry/otlp/OpenTelemetryOtlpHttp4sGrpcSpanExporter.scala

```diff
@@ -0,0 +1,35 @@
+package trace4cats.opentelemetry.otlp
+
+import cats.Foldable
+import cats.effect.kernel.Temporal
+import cats.syntax.all._
+import io.opentelemetry.proto.collector.trace.v1.service.TraceService
+import org.http4s.client.Client
+import org.http4s.{Header, Headers, Uri}
+import trace4cats.kernel.SpanExporter
+import trace4cats.model.Batch
+import trace4cats.opentelemetry.otlp.proto.ExportSpansRequest
+
+object OpenTelemetryOtlpHttp4sGrpcSpanExporter {
```

Comment:
Could you also mirror the host/port constructor like [here](https://github.com/trace4cats/trace4cats-opentelemetry/blob/master/modules/opentelemetry-otlp-grpc-exporter/src/main/scala/trace4cats/opentelemetry/otlp/OpenTelemetryOtlpGrpcSpanExporter.scala#L12-L17) please?